tests-php/Nominatim/NominatimTest.php

```diff
@@ -1,7 +1,7 @@
 <?php
 
 namespace Nominatim;
-require 'lib/lib.php';
+require '../lib/lib.php';
```

Comment:
This does not work for me. As the phpunit.xml is in the root directory, I also have to run phptest from the root directory. The README in test-php seems to be wrong on that subject or maybe I'm running the tests in the wrong way.